### ISSUE TITLE
Odd warning in guide.py after #105

### ISSUE THREAD

**Issue by taldcroft** [sot/proseco#120]
After including #105 I am seeing a warning I don't understand for one obsid:
```
/Users/aldcroft/git/proseco/proseco/guide.py:416: RuntimeWarning: invalid value encountered in sqrt
  stars['MAG_ACA_ERR'][~stars['offchip']][pos_spoil] ** 2))
/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/astropy/table/column.py:965: RuntimeWarning: invalid value encountered in greater
  return getattr(self.data, op)(other)
```
This came up in bulk processing of every obsid in 2018, unfortunately at the moment the processing is not showing the obsid.  But I just restarted with a print(obsid).

**Comment by taldcroft** [comment]
Obsid 20515

**Comment by jeanconn** [comment]
I think the bigger problem is that plugging in MAG_ACA_ERR was fine conceptually, but since I was used to work with a magerr that was in mag units, I forgot to convert MAG_ACA_ERR from 0.01 mag units.

**Comment by taldcroft** [comment]
- I'm surprised existing tests didn't catch this.  You should add a catalog (aca-level) test of a super-dense field for regression, where with a mag err problem most candidates would be spoiled (with the mistake).  This is a good thing for fid and acq selection as well.
- I didn't catch this in review, :frowning:

**Comment by jeanconn** [comment]
Right, the unit test problem on this was just that spoiled just marks spoiled, not by how much (without the 0.01mag multiplier on error... super duper spoiled).

**Comment by jeanconn** [comment]
And I just confirmed that #123 does get rid of the reproducible "RuntimeWarning: invalid value encountered in sqrt" error on obsid 20515.  So this will be closed by #123
